src/TouristInfoCards.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+const TouristInfoCards = props => {
+  return (
+    <div className="cardsContainer">
+      {props.cities.map(city => {
+        return (
+          <div className="card">
+            <img
```

Comment:
I think I have suggested extracting lines 8 to 21 to a separate `Card` component in my previous review?🙂